### PR TITLE
Feat/incidents list

### PR DESCRIPTION
**Pull request by JIamep20**

### CODE REVIEW

File: src/components/Incidents/IncidentList/index.js [hunk 1]
```diff
@@ -1,29 +1,146 @@
...
+    >
+      <td
+        colSpan={2}
+        className="px-6 py-4 text-darker-black col-span-2 text-sm leading-5 font-medium"
+      >
+        {title}
+      </td>
+      <td className="px-3 py-3">
+        <div className="flex flex-row items-center">
+          <Icon name={severityInfo.icon} />
```
Comment (JIamep20) [inline src/components/Incidents/IncidentList/index.js:109]: i decided this is more universal. heroicons may not contain all the required icons

Comment (moshloop) [reply src/components/Incidents/IncidentList/index.js:109]: We use https://react-icons.github.io which has pretty much all free icon sets - The included icons are intended primarily for product logos.

Comment (JIamep20) [reply src/components/Incidents/IncidentList/index.js]: @moshloop done. Anything else here ?

File: src/components/Incidents/IncidentList/index.js [hunk 2]
```diff
@@ -1,29 +1,145 @@
-import { Link } from "react-router-dom";
+import React, { useMemo } from "react";
+import { useNavigate } from "react-router-dom";
+import cx from "clsx";
```
Comment (moshloop) [inline src/components/Incidents/IncidentList/index.js:3]: why the rename from clsx to cx ?

**Comment by moshloop** [comment]
merged into v2 branch already

**Pull request closed without merging**